### PR TITLE
[NoTicket] Update to latest json schema

### PR DESCRIPTION
**Pull request by StephenBlackWasAlreadyTaken**
We need support for oneOf which does not exist in our fork!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog JSON Schema Tools
 
+##2015-04
+
+Thanks @VidurMalik
```
Comment (StephenBlackWasAlreadyTaken) [inline CHANGELOG.md:5]: @VidurMalik hey I know that guy

**Comment by krames** [comment]
@VidurMalik can I get your :eyes: on this?

**Comment by VidurMalik** [comment]
This is basically pulling in changes from the repo, as far as I can see. :shipit: :shipit: :shipit:

**Comment by bfosberry** [comment]
:+1:  should not even think twice

have you guys tested this with the SK codebase?

**Comment by krames** [comment]
Thanks guys! We are planning on testing that today.

**Pull request merged**